eureka2-core/src/main/java/com/netflix/eureka2/channel/AbstractServiceChannel.java

```diff
@@ -60,4 +66,27 @@ public void onNext(T message) {
             }
         });
     }
+
+    protected boolean moveToState(STATE from, STATE to) {
```

Comment:
How does this work with the (now) overridden moveToState in the specific channelImpls?